comms/dht/Cargo.toml

```diff
@@ -23,10 +23,9 @@ bytes = "0.4.12"
 chacha20 = "0.7.1"
 chrono = "0.4.9"
 diesel = {version="1.4.7", features = ["sqlite", "serde_json", "chrono", "numeric"]}
-diesel_migrations =  "1.4"
+diesel_migrations = "1.4.0"
 digest = "0.9.0"
 futures= {version= "^0.3.1"}
```

Comment:
Bundling must be put behind a feature flag so that it is bundled for all apps except libwallet

Reply:
Made libsqlite3-sys an optional dependency and filtered it out in wallet_ffi.